resources.whatwg.org/standard.css

```diff
@@ -368,12 +368,12 @@ td.non-rectangular-cell-indentation { border-top-style: hidden; min-width: 2em;
 .hide { display: none }
 
 .head { margin: 0 0 1em; padding: 1em 0 0 0; display: block; }
-.head p { margin: 0; }
+.head p:not(hgroup p) { margin: 0; }
 .head h1 { margin: 0 100px 0 0; }
-hgroup p { margin: 0 100px 1em 0; }
+hgroup p, .head h2 { margin: 0 100px 1em 0; }
```

Comment:
Does `.head h2` still exist then?

Reply:
Hmm, seems unlikely, not sure why I did that...